lua/telescope/_extensions/file_browser/finders.lua

```diff
@@ -32,7 +32,9 @@ fb_finders.browse_files = function(opts)
       table.insert(data, typ == "directory" and (entry .. os_sep) or entry)
     end,
   })
-  table.insert(data, 1, ".." .. os_sep)
+  if opts.cwd ~= os_sep then
```

Comment:
Sorry, my bad in the diff. This should be `opts.path`.

Reply:
lol! Sorry - we'll get there! :)